filesystem/filesystem.go

```diff
@@ -18,6 +18,8 @@ type FileSystem interface {
 	Mkdir(path *fspath.FileSystemPath, workingDir file.File) (file.File, error)
 	MkdirAll(path *fspath.FileSystemPath, workingDir file.File) (file.File, error)
 	CreateRegularFile(path *fspath.FileSystemPath, workingDir file.File) (file.File, error)
+	DefaultWorkingDirectory() file.File
+	ChangeWorkingDirectory(path *fspath.FileSystemPath, workingDir file.File) (file.File, error)
```

Comment:
This should probably be renamed something like GetDirectory. Changing the working directory will be managed by the client/process using the library